components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/internal/FeatureManifestInterface.kt

```diff
@@ -41,4 +41,9 @@ interface FeatureManifestInterface<T> {
      * is quite useful for introspection.
      */
     fun getFeature(featureId: String): FeatureHolder<*>?
+
+    /**
+     * Get a list of feature ids where the feature allows co-enrollment.
+     */
+    fun getCoenrolledFeatureIds(): List<String>? = null
```

Comment:
I don't think this should be optional. You can always use `listOf()` if you want an empty list.